doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -24,33 +24,99 @@ mvn install
 ----
 +
 This command builds the KJAR file and places it into the local Maven repository. By default, this repository is located in the `~/.m2/repo` directory.
-. In the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` directory, add the following properties to your Spring Boot application `application.properties` file:
+. In the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` directory, add the following property to your Spring Boot application `application.properties` file:
 +
 [source]
 ----
 kieserver.classPathContainer=true
+----
++
+When this property is set to `true`, {KIE_SERVER} uses the class loader used by the container to load KJAR files and their dependencies.
++
+. Complete one of the following actions to ensure that the {KIE_SERVER} loads the necessary KJAR modules:
+** Add the following property to the `application.properties` file:
```

Comment:
I like the solution you came up with. Just a few suggestions.

For each bullet point here, start with telling me in what situation I would choose a particular bullet. I know you explain in the text below the first bullet, but some link this:

To set KIE Server to scan and deploy all KJAR modules available in the Spring Boot application, Add the following property to the application.properties file:.
And then modify the section I stole that from so it's not too redundant.